resources/arduino_files/power_board/power_board.ino

```diff
@@ -84,6 +84,14 @@ void setup() {
   
   digitalWrite(DSEL_0, LOW); // DSEL_0 LOW reads PROFET 0_0. DSEL_0 HIGH reades PROFET 0_1
   digitalWrite(DSEL_1, LOW); // DSEL_1 LOW reads PROFET 1_0. DSEL_1 HIGH reades PROFET 1_1  
+
+  // Turn on everything to start
```

Comment:
Given the lack of pull up resistors, I'd be inclined to turn the relationship on as soon as possible